lib/Listener/ChattyLLMTaskListener.php

```diff
@@ -52,17 +52,30 @@ public function handle(Event $event): void {
 		}
 
 		// message generation
-		if (preg_match('/^chatty-llm:(\d+)$/', $customId, $matches)) {
+		if (preg_match('/^chatty-llm:(\d+)/', $customId, $matches)) {
 			$sessionId = (int)$matches[1];
 
 			$message = new Message();
 			$message->setSessionId($sessionId);
 			$message->setOcpTaskId($task->getId());
 			$message->setRole('assistant');
-			$message->setContent(trim($task->getOutput()['output'] ?? ''));
 			$message->setTimestamp(time());
 			$sources = json_encode($task->getOutput()['sources'] ?? []);
-			$message->setSources($sources ? $sources : '[]');
+			$message->setSources($sources ?: '[]');
+			if (class_exists('OCP\\TaskProcessing\\TaskTypes\\AudioToAudioChat')
+				&& $taskTypeId === \OCP\TaskProcessing\TaskTypes\AudioToAudioChat::ID) {
+				$message->setContent(trim($task->getOutput()['output_transcript'] ?? ''));
+				$message->setAttachments('[{"type":"Audio","fileId":' . $task->getOutput()['output'] . '}]');
```

Comment:
The output file is deleted by the task processing cleanup job after a while. Do we want to make sure that it lives longer than that?